qtism/common/utils/Format.php

```diff
@@ -355,7 +355,7 @@ static public function isString256($string) {
 	 * @return boolean
 	 */
 	static public function isClass($string) {
-	    $pattern = "/^(?:[^\s]+?(?:\x20){0,1})+$/";
+        $pattern = "/^(?:[^\s]+?(?:\x20){0,})+$/";
```

Comment:
The regex looks fine as such a case is still compatible with the qti xsd (xsd validation tested during import)